.github/workflows/pr-ci.yml

```diff
@@ -150,9 +169,66 @@ jobs:
 
           To run smoketest:
           ```
-          # amd64          
+          # amd64
           CRYOSTAT_IMAGE=${{ env.amd64_image }} sh smoketest.sh
 
           # or arm64
           CRYOSTAT_IMAGE=${{ env.arm64_image }} sh smoketest.sh
           ```
+
+  integration-test:
+    needs: [checkout-branch, push-to-ghcr]
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    uses: ./.github/workflows/integrated-test.yml
+    with:
+      build-arch: ${{ matrix.arch }}
+      pr-number: ${{ github.event.issue.number }}
+      sha-value: ${{ needs.checkout-branch.outputs.PR_head_sha }}
+  
+  retest-integration:
+    needs: [checkout-branch]
+    if: contains(github.event.comment.body, '/retest')
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    uses: ./.github/workflows/integrated-test.yml
+    with:
+      build-arch: ${{ matrix.arch }}
+      pr-number: ${{ github.event.issue.number }}
+      sha-value: ${{ needs.checkout-branch.outputs.PR_head_sha }}
+      
+  integration-test-pass:
```

Comment:
this can be moved into the `integration-test.yml` file and added on as the last step of the `integration-tests` job? Then you wont need separate jobs here for `integration-test-pass` and `retest-integration-pass`

Reply:
> this can be moved into the `integration-test.yml` file and added on as the last step of the `integration-tests` job? Then you wont need separate jobs here for `integration-test-pass` and `retest-integration-pass`

The idea was to produce different outputs depending on the test